packages/web-react/src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -30,13 +30,13 @@ export const Breadcrumbs = <T extends ElementType = 'nav'>(props: SpiritBreadcru
           items?.map((item, index) => (
             <React.Fragment key={`BreadcrumbsItem_${item.title}`}>
               {index === items.length - 2 && goBackTitle && (
-                <BreadcrumbsItem href={item.url} iconNameStart="chevron-left" isGoBackOnly>
+                <BreadcrumbsItem href={item.url || undefined} iconNameStart="chevron-left" isGoBackOnly>
```

Comment:
I think this is not a good solution because there will be still the `Link` component, but with empty `href`. And I think we do not want to have an `a` without `href` in rendered HTML, do we?

Reply:
But it does not render an `a` without `href` it renders as plain text as expected. So I would say it is ok?